src/extension/kernel.ts

```diff
@@ -184,6 +185,14 @@ export class Kernel implements Disposable {
     this.category = category
   }
 
+  setSerializer(serializer: GrpcSerializer) {
+    this.serializer = serializer
+  }
+
+  getSerializer() {
```

Comment:
Interface not narrow enough, please see comment inside where saving is handled

Reply:
Fixed in 2c69b84a17dcf327bd3ee5e4e305200fb220cece